developer_guides/unit_tests.rst

```diff
@@ -84,6 +84,30 @@ Preparing cmocka package
 
       ./configure --with-cmocka-prefix=/home/<you>/cminstall ...
 
+Wraping objects for unit tests
+******************************
+
+If at any point should You need to mock any symbol it can be done by simply defining it in unit test and only including .h files. However there can exist corner cases in which function that should be mocked can't be done in this way. The cases are - Static functions in headers(those most probably are inline short functions and don't have to be mocked) or static functions that are in the same file as tested functionality and are exceedingly large so they can't be tested as one functionality. Whatever the reason mocking of those symbols can be done by using --wrap linker functionality. To wrap the symbol follow these steps:
+
+#. Create mocked symbol named __wrap_symbol_name
+
+#. Pass instruction for the linker -Wl, --wrap=symbol_name during compilation. Now every symbol call to symbol_name will call __wrap_symbol_name.
+
+Passing instruction to linker in SOF UT enviroment can be done by use of CFLAGS. However this is a subject to change since linker flags should be passed in separate variable in makefiles.

```

Comment:
Line 96 - "Instructions can be passed to the linker in the SOF UT environment using CFLAGS, however they should be passed in separate variables in the makefile."